scripts/install.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+if [ -z "$1" ]
+  then
+    echo "Please specify the OS distribution of the node"
+fi
+nodeos="$1"
+kubectl apply -f ../install/falco_loader/$nodeos/falco_loader_pod.yaml
+if [ $? -ne 0 ]; then
+  exit 1
+fi
+echo "Waiting for falco loader pod to complete. This might take 6 mins..."
+kubectl wait --for=condition=Completed pod/falco-loader --timeout=360s
+if [ $? -ne 0 ]; then
+  exit 1
+fi
+
+echo "Starting syscalls tracer..."
+kubectl create ns kubesecgen
+kubectl apply -f ../install/clusterrole.yaml
```

Comment:
nit: if you want to install all the yaml files in the directory you can simply do `kubectl apply -f ../install`